src/lib/containers/drug_upload_tool/DrugUploadTool.tsx

```diff
@@ -337,11 +337,24 @@ class DrugUploadTool extends React.Component<
     }
   };
 
+  renderUnauthenticatedView = (token: string | undefined) => {
+    if (token) {
+      return <></>
+    } else {
+      return (
```

Comment:
I believe that this should change slightly from renderUnauthenticatedView -> renderUnauthorizedView, and the check would be if (state.status.UNAUTHORIZED) {} because the portal will block the user from reaching this component until they signed in and have a token